static/app/views/explore/components/seerComboBox/seerSearchSkeleton.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from '@emotion/styled';
+
+import {t} from 'sentry/locale';
+import {SeerSearchHeader} from 'sentry/views/explore/components/seerComboBox/seerSearchHeader';
+
+function generateThreeUniqueNumbers(): number[] {
+  const numbers: Set<number> = new Set();
+  const min = 35;
+  const max = 95;
+
+  while (numbers.size < 3) {
+    numbers.add(Math.floor(Math.random() * (max - min + 1)) + min);
+  }
+
+  return Array.from(numbers);
+}
+
+export function SeerSearchSkeleton() {
+  const numbers = generateThreeUniqueNumbers();
```

Comment:
Does this need to be generated uniquely every time? 

It also might be better to put in is a useMemo so that you don't have the risk of it rerendering and changing size